### ISSUE TITLE
Support Watch window 'Break when value it true' and 'Break when value changes'

### ISSUE THREAD

**Issue by WaynePhillipsEA** [WaynePhillipsEA/twinbasic#76]
**Describe the solution you'd like**
As per title, VBA and VB6 allow watch expressions to have the 'break when value is true' or 'break when value changes' options set on them, which can be handy for debugging.

**Additional context**
This feature requires evaluating each flagged expression after every single line of code execution, causing a performance concern.  However, this debugging feature is used by many VBA/VB6 developers and so twinBASIC should also support it.

**Comment by wqweto** [comment]
Watch expressions usually have a very limited scope -- within a procedure or a module -- so the performance hit is mitigated somewhat.

**Comment by WaynePhillipsEA** [comment]
That's true, and is something that we should consider when implementing this feature.

**Comment by retailcoder** [comment]
Isn't this already supported?

![image](https://user-images.githubusercontent.com/5751684/115350586-4a0bb100-a183-11eb-981d-f59cba5f1be0.png)

![image](https://user-images.githubusercontent.com/5751684/115350643-5c85ea80-a183-11eb-80af-4536a907eb21.png)

![image](https://user-images.githubusercontent.com/5751684/115350687-69a2d980-a183-11eb-8c14-f99ade670eca.png)

Breakpoint hit:

![image](https://user-images.githubusercontent.com/5751684/115350735-76bfc880-a183-11eb-9c94-e743eeefff43.png)

**Comment by WaynePhillipsEA** [comment]
@retailcoder I think that is not working as expected, but being hit unconditionally

**Comment by retailcoder** [comment]
Ooh, makes sense - you need to hook it up to a service that can evaluate the expression! Good to know!
